03.macos/05.solving-problems/01.install-cert/docs.ru.md

```diff
@@ -18,6 +18,6 @@ visible: true
   
   4. Перйдите во вкладку "Центры сертификации" и кликните по кнопке "Импортировать..."
   
-  5. Выберите файл `/Library/Application Support/com.adguard.Adguard/NfApiConfiguration/SSL/Adguard Perosnal CA.cer` или скачайте его через любой браузер на основе Chromium (например, Google Chrome или новый Edge) с запущенной HTTPS-фильтрацией в AdGuard
+  5. Выберите файл `/Library/Application Support/com.adguard.Adguard/AdguardCore/Adguard Perosnal CA.cer` или скачайте его через любой браузер на основе Chromium (например, Google Chrome или новый Edge) с запущенной HTTPS-фильтрацией в AdGuard
```

Comment:
File path is `/Library/Application Support/com.adguard.mac.adguard/AdguardCore/Adguard Personal CA.cer` in fact, or `/Library/Application Support/com.adguard.mac.adguard-setapp/AdguardCore/Adguard Personal CA.cer` for _setapp_ application version